doc/rippled-example.cfg

```diff
@@ -609,6 +609,15 @@
 #
 #
 #
+# [validator_key_revocation]
+#
+#   If a validator's keys have been compromised, a revocation must be generated
```

Comment:
Since revocations are expected to have multiple lines, only one `[validator_key_revocation]` can currently be configured.
```
[validator_key_revocation]
JP////9xIe0RORU7epDJDafo+X+cRnRfyDyz/eIZ6kx4r6+mfnw0SXASQC6zw2s75NkhxHSY
5N5cniQ90MRADk+/xVRwry2QrqDS561sukMtXqwWdoNO0IMMnh9XmL5z/WHs8peRkBWW6AE=
```
We can either require that a revocation be confined to a single line to allow multiple, or effectively communicate that if a second set of validator keys are compromised, the new revocation must replace the old one in the config.